### PR TITLE
Fix tuple encoding bug

### PR DESCRIPTION
**Pull request by satyamakgec**
Not anymore encoding the dynamic length in the output when the return type is tuple value.

resolve this - https://github.com/weiroll/weiroll/issues/93

### CODE REVIEW

**Review by Arachnid — APPROVED** [review]

File: src/planner.ts [hunk 1]
```diff
@@ -743,7 +743,7 @@ export class Planner {
           isDynamicType(command.call.fragment.outputs?.[0]) ||
           (command.call.flags & CommandFlags.TUPLE_RETURN) !== 0
         ) {
-          ret |= 0x80;
+          ret |= 0x00;
```
Comment (Arachnid) [inline src/planner.ts:746]: You can delete this line entirely.

**Review by Arachnid — APPROVED** [review]

File: src/planner.ts (hunk 1 above)
Comment (satyamakgec) [reply src/planner.ts:746]: So that means completely removing the if block?

Comment (Arachnid) [reply src/planner.ts:746]: Yes.

Comment (satyamakgec) [reply src/planner.ts:746]: I think it is important to keep the `if` loop for the dynamic return values like string, array and whatnot so the MSB get set and in the case of tuple we don't need it so we can remove the `or` condition.

So the new change will look like this - 

```
if (isDynamicType(command.call.fragment.outputs?.[0]) ) {
          ret |= 0x80;
}
```

**Comment by satyamakgec** [comment]
@Arachnid can we please merge this and publish the new package of it?

**Review by Arachnid — APPROVED** [review]

**Pull request merged**